user_guide/admin_capabilities.rst

```diff
@@ -0,0 +1,464 @@
+
+
+.. _admin-capabilities-search:
+
+Search
+------
+
+If the capability is set to *"System"*, the user can use the :ref:`search <user-guide-getting-started-search>` 
+functionality to quickly find specific records.
+
+This is a general capability that can improve the overall experience of all users.
+
+The setting does not influence the user's ability to :ref:`search by tag <user-guide-getting-started-search-tag>`.
+
+
+.. _admin-capabilities-campaign-emails:
+
+Send Campaign Emails
+--------------------
+
+With OroCRM's :ref:`email campaigns <user-guide-email-campaigns>`, the user can send personalized template-based emails 
+to multiple users. This capability doesn't effect the users' ability to define and edit the campaign settings and create 
+templaets, but on if this capability is set to *"System"*, the user can launch the camplaign, i.e. start 
+:ref:`sending emails <user-guide-email-campaigns-send>` specified by the campaign.
+
+Marketing associates and other employees authorized for direct communications with potential 
+customers, existing clients, other system users, etc. are usually authorized to send email campaigns.
+
+|
+
+.. image:: ./img/roles/email_campaign.png
+
+|
+
+
+.. _admin-capabilities-share-grid:
+
+Share Grid View
+---------------
+
+If this capability is set to *"System"*, the user can share the :ref:`grid <user-guide-ui-components-grids>` views 
+that they have configured, this way the user can :ref:`adjust a grid <user-guide-ui-components-grids-adjust>` and share 
+it with other users.
+
+This is particularly useful for team-leads and heads of departments, to modify and share grids with their subordinates. 
+
+|
+
+.. image:: ./img/roles/grid_share.png
+
+|
+
+ 
+.. _admin-capabilities-system-info:
+
+System Information
+------------------
+
+If this capability is set to *"System"*, the user can view the system information page. This page contains the list of 
+Oro packages and third-party packages that are installed, and is usually only used by system administrators and 
+integrators.
+
+.. _admin-capabilities-system-config:
+
+System Configuration
+--------------------
+
+If this capability is set to *"System"*, the user can access the :ref:`system configuration page <admin-configuration>`
+to localize the system, change the display and tracking settings, and otherwise change the system configuration.
+
+
+.. _admin-capabilities-tags:
+
+Tag Assign/Unassign
+-------------------
+
+If this capability is set to *"System"*, the user can  assign/unassign :ref:`tags <user-guide-tags>` which are 
+non-hierarchical keywords or phrases assigned to records. They provide additional information about records and
+are visible to all the system users. 
+
+Tags can be successfully utilized by all the users.
+
+
+.. _admin-capabilities-tags-all:
+
+Unassign All Tags From Entities
+-------------------------------
+
+This capability is only meaningful if *"Tag assign/unassign"* is set to *"System"*.
```

Comment:
Edit: his capability is only meaningful if *"Tag assign/unassign"* is set to *"System."*